mission_control/navigator_launch/launch/subsystems/sensors.launch

```diff
@@ -7,4 +7,7 @@
 
   <!-- Sick -->
   <include file="$(find navigator_launch)/launch/subsystems/sick_lidar.launch"/>
+  
+  <!-- Host Monitoring -->
+  <node name="host_monitor" pkg="navigator_gui" type="host_monitor.py">
```

Comment:
Does only one person usually run `master_shore.launch`? Multiple people trying to run the node would be unnecessary. This makes sense if that is the case.